services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        return movie_session.filter(show_time__date=session_date)
+
+    return movie_session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int = None,
+                         show_time: str = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> None:
+    update_session = MovieSession.objects.filter(id=session_id)
+
+    if show_time:
+        update_session.update(show_time=show_time)
+    if movie_id:
+        update_session.update(movie_id=movie_id)
+    if cinema_hall_id:
+        update_session.update(cinema_hall_id=cinema_hall_id)
+
+
+def delete_movie_session_by_id(session_id: int) -> None:
+    MovieSession.objects.filter(id=session_id).delete()
```

Comment:
read the 3 item here
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency